### PR TITLE
Show issue title at dashboard

### LINKED ISSUE #1854: Show issue title for comment news feeds
it could go just under the header line:

**username commented on issue organisation/project-name#221**
*Need to process the input using sql queries directly*
"Comment text goes here"

This is because with so many issues, it can become a little confusing what issue number refers to what issue!

### PR DESCRIPTION
**Pull request by makhov**
Resolve #1854

### CODE REVIEW

File: models/action.go [hunk 1]
```diff
@@ -136,6 +137,12 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID, _ := strconv.Atoi(strings.SplitN(a.Content, "|", 2)[0])
```
Comment (Unknwon) [inline models/action.go]: You can import `github.com/Unknwon/com` and use `com.StrTo().MustInt64`.

Also,

```go
issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
```

**Comment by makhov** [comment]
update pr

File: models/action.go [hunk 2]
```diff
@@ -136,6 +137,12 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
+	issue, _ := GetIssueByID(issueID)
```
Comment (Unknwon) [inline models/action.go]: Should we check error here?

`if err !=nil` then `log.Error(4, "GetIssueByID: %v", err)` and returns somethings like `500 when get title` as title?

Comment (makhov) [inline models/action.go]: Fix it again

File: models/action.go [hunk 3]
```diff
@@ -136,6 +137,16 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
```
Comment (Unknwon) [inline models/action.go]: Thanks. Now the question is index `0` is actually the index of issue, not the title...

Comment (makhov) [inline models/action.go]: As I understood index ```0``` is the issue id. At the next line I call ```GetIssueById```.
What's wrong?

Comment (Unknwon) [inline models/action.go]: Oops, my bad, I really mean it is the issue index, not issue ID.

Comment (Unknwon) [inline models/action.go]: But I know how to fix it, you can leave it to me or I can indicate where changes should be made. Which way you prefer?

Comment (makhov) [inline models/action.go]: I see. Please, look at last commit.

**Comment by makhov** [comment]
Change Id to Issue

**Comment by Unknwon** [comment]
It should work now, thanks!

**Pull request merged**